Source/WebKit/Shared/ProcessQualified.serialization.in

```diff
@@ -94,6 +97,13 @@ header: <WebCore/ProcessQualified.h>
     WebCore::ProcessIdentifier processIdentifier();
 };
 
+#if ENABLE(THREADED_ANIMATION_RESOLUTION)
+[Alias=class ProcessQualified<TimelineIdentifierID>, AdditionalEncoder=StreamConnectionEncoder, CustomHeader] alias WebCore::TimelineIdentifier {
```

Comment:
I'd prefer if we just serialised the non-process-qualified identifier, and then converted to the process qualified version upon receipt of the data (using trusted data about which process the IPC message came from).

This prevents a rogue web process being able to lie about their process identifier, and thus control animations from a different iframe.

That said, this follows the existing approach for layer identifiers (which are hard to fix on a technical level), so it's probably fine to be consistent.